ROADMAP.md

```diff
@@ -107,41 +109,41 @@ Status | Command | Description
 
 ## Pattern searches
 
-Status | Command | Description
----|--------|------------------------------
-:warning: | :1234: `/{pattern}[/[offset]]<CR>` | search forward for the Nth occurrence of {pattern}
-:warning: | :1234: `?{pattern}[?[offset]]<CR>` | search backward for the Nth occurrence of {pattern}
- | :1234: `/<CR>` | repeat last search, in the forward direction
- | :1234: `?<CR>` | repeat last search, in the backward direction
-:warning: | :1234: n | repeat last search
-:warning: | :1234: N | repeat last search, in opposite direction
+Status | Command | Description | Note
+---|--------|------------|------------------
+:warning: | :1234: `/{pattern}[/[offset]]<CR>` | search forward for the Nth occurrence of {pattern} | Currently we only support JavaScript Regex but not Vim's in-house Regex engine.
+:warning: | :1234: `?{pattern}[?[offset]]<CR>` | search backward for the Nth occurrence of {pattern} | Currently we only support JavaScript Regex but not Vim's in-house Regex engine.
+:white_check_mark: | `/<CR>` | repeat last search, in the forward direction
```

Comment:
sorry, this is actually half implemented, it is supposed to have the num option that you removed.

Both this and the following are implemented for the non num preceded version so maybe yellow hazard icon?